all-actors/src/main/java/org/sunbird/CertVars.java

```diff
@@ -37,4 +38,11 @@ private static String getPropsFromEnvs(String props){
         String propValue=System.getenv(props);
         return propValue;
     }
+
+
+    public static String getEsSearchUri(){
+        String esApi=String.format("http://%s:9200/%s/_search",getPropsFromEnvs(SUNBIRD_ES_IP).split(",")[0],JsonKeys.CERT);
```

Comment:
lets not assume port number. Please modify and get the entire host path from env var. We will only append _search or _doc as the case maybe (for SEARCH and READ).

Reply:
@indrajra 
sunbird_es_port=9300 this value is been set in env,  but rest calls needs to be communicated to 9200 port only so that's why it is hardcoded.